javascripts/discourse/templates/components/dc-categories-only.hbs

```diff
@@ -0,0 +1,47 @@
+{{!-- TODO: https://github.com/debtcollective/community/issues/32 --}}
+{{#if categories}}
```

Comment:
@duranmla I see how adding the `dc` prefix in classes makes sense, but I'm not sure about filenames. What's the reason behind it?

Reply:
Because otherwise we will have the same file name of discourse and we won't achieve what we want _(create a custom template to use in mobile and desktop)_

The other option is to have a different name, but I like the fact of being able to know which custom template belongs to which discourse template _(maybe in other cases doesn't makes sense but in this one it does IMHO)_

Reply:
I think is possible to have templates for both mobile and desktop by overriding the the files. What about having a call to go through this. This will affect how we implement other things

Reply:
Let's add a session today in the calendar to talk about it. If you can search beforehand what's your proposal will be great. I will continue with couple extra details we need to implement in the mean time